test/sslapitest.c

```diff
@@ -12013,7 +12109,12 @@ int setup_tests(void)
             OSSL_PROVIDER_unload(prov);
         }
     }
-
+#if 0
+#ifndef OPENSSL_NO_USABLE_ECH
+    if (!is_fips && hpke_setlibctx(libctx)!=1)
```

Comment:
nit: space around `!=`

Reply:
That's all under a ``#if 0`` so I just took it out. (Probably a remnant from the HPKE PR which originated in this code too.)